app/views/dresses/listings.html.erb

```diff
@@ -23,8 +23,9 @@
       <p><%= dress.description %></p>
       <p>$<%= dress.price_per_day %> per day</p>
       <div class="edit-dress">
-        <%= link_to "Delete", dress_path(dress), method: :delete, data: {confirm: "Are you sure?"}, class: "btn btn-outline-secondary mb-5 mx-1" %>
+        <%= link_to "", dress_path(dress), method: :delete, id: "delete-link", class: "hidden" %>
```

Comment:
Does the button still say delete?

Reply:
This "link" is actually hidden and the "button" below is shown. The visible button pops up the sweetalert. When the user confirms, an event listener clicks on this hidden delete "link" to actually process the delete method.

Reply:
That's cool, thanks for implementing this.